DiscordBot.DataAccess/EventsSheetsService.cs

```diff
@@ -32,7 +31,7 @@ public class EventsSheetsService : IEventsSheetsService
     {
         private static readonly string[] scopes = { SheetsService.Scope.Spreadsheets };
         private static readonly string applicationName = "Softwire Discord Bot";
-        private static readonly string spreadsheetId = "";
+        private static readonly string spreadsheetId = Environment.GetEnvironmentVariable("GOOGLE_SHEET_ID");
```

Comment:
I'd recommend storing the names of these variables somewhere, e.g. a static class with several const strings 🙂

Reply:
Done in 4d9b757 🙂

An alternate solution that I came up with was to extract all of the `Environment.GetEnvironmentVariable(...)` out into a static class, along the line of:

<details>
<summary>Spoiler</summary>

```C#
internal static class SheetsEnvironmentVariables
{
    const string sheetIdVar = "GOOGLE_SHEET_ID";
    const string clientEmailVar = "GOOGLE_CLIENT_EMAIL";
    const string projectIdVar = "GOOGLE_PROJECT_ID";
    const string privateKeyIdVar = "GOOGLE_PRIVATE_KEY_ID";
    const string privateKeyVar = "GOOGLE_PRIVATE_KEY";

    public static string? SheetId => Environment.GetEnvironmentVariable(sheetIdVar);
    public static string? ClientEmail => Environment.GetEnvironmentVariable(clientEmailVar);
    public static string? ProjectId => Environment.GetEnvironmentVariable(projectIdVar);
    public static string? PrivateKeyId => Environment.GetEnvironmentVariable(privateKeyIdVar);
    public static string? PrivateKey => Environment.GetEnvironmentVariable(privateKeyVar);
}
```
</details>

What do you think? 🙂

Reply:
I like it 🙂

Reply:
Added in 427658b 🙂